In [27]:
Input: 20
    Output:
         19KQ1A0501
            | | | | |
            19KQ1A0509
            19KQ1A0510

IndentationError: unexpected indent (<ipython-input-27-4392ccc3a0e0>, line 2)

In [28]:
n = int(input())
for j in range(n):
    print("19AK1A05{:02}".format(j))

11
19AK1A0500
19AK1A0501
19AK1A0502
19AK1A0503
19AK1A0504
19AK1A0505
19AK1A0506
19AK1A0507
19AK1A0508
19AK1A0509
19AK1A0510


In [ ]:
{ } -> .format(?)
{:01} -> .format(?)
{:02} -> .format()

## while -> Unknown ranges, Known ranges
        syntanx:
              variable init
              while(condition):
                   //stmnt
                   // incr/decr

In [48]:
n = int(input())
while n!=10:
      print(n,end='' '')
    n= n-1

IndentationError: unindent does not match any outer indentation level (<tokenize>, line 4)

In [50]:
n = int(input())
print(n//10)

1259
125



 Input: 1259
      output:
        Given number is:1259
        reverse of a number is:19521
        Given number is 1259 and its reverse is:9521   

In [56]:
n = int(input())
r = 0
p = n
while n!=0:
    r = r*10+n%10
    n //=10
print("Given number is : {}\nReverse of a number is :  {}\n ""Given number is {} and its reverse is : {}".format(p,r,p,r))

1259
Given number is : 1259
Reverse of a number is :  9521
 Given number is 1259 and its reverse is : 9521


## Jumping statements:
                 - Break -> exit
                 - Continue -> skip
                 - Pass -> skip nothing will be printed
                 - Return -> values to be sent for function

In [57]:
n = int(input())
for k in range(n + 1):
    print(k,end=",")

10
0,1,2,3,4,5,6,7,8,9,10,

In [67]:
n = int(input())
for h in range (n):
    if h == 5:
        break
    else:
            print(h,end=" ")
print("\n",h)

100
0 1 2 3 4 
 5


In [66]:
n = int(input())
for h in range (n):
    if h == 5:
        continue
    else:
            print(h,end=" ")
print("\n",h)

20
0 1 2 3 4 6 7 8 9 10 11 12 13 14 15 16 17 18 19 
 19


In [70]:
n = int(input())
for b in range (n):
    if b == 10:
        pass
    else:
            print(b,end=" ")

20
0 1 2 3 4 5 6 7 8 9 11 12 13 14 15 16 17 18 19 

In [87]:
for i in 'hello':
    if(i == 'e'):
        print('pass executed')
        pass
    print(i)
    print(' ----')

for i in 'hello':
    if(i == 'e'):
        print('continue executed')
    continue
    print(i)

h
 ----
pass executed
e
 ----
l
 ----
l
 ----
o
 ----
continue executed


## Inner Loops:
                - Loop within a Loop
             syntax:
                     for i in range():
                        for j in range():
                              if (conditional-n):
                                   //stmnts
                              else:
                                   //stmnts
                               print("\n")

 1 - 31
      r -  5
      c -  7

     step-1
    01 02 03 04 05 06 07
    08 09 10 11 12 13 14
    15 16 17 18 19 20 21
    22 23 24 25 26 27 28 
    29 30 31
    
    step-2
      01 **03 ** 05 ** 07
    ** 09 **11 ** 13 **
    15 ** 17 ** 19 ** 21
    ** 23 ** 25 ** 27 ** 
    29 ** 31
    
    step-3
   
      01 **03 ** ## ** 07
    ** 09 **11 ** 13 **
    15 ** 17 ** 19 ** 21
    ** 23 ** 25 ** 27 ** 
    29 ## 31

In [113]:
a = 1
for i in range(1,6):
    for j in range(1,8):
            if  a <= 31:
                if a%5==0:
                    print("##",end=" ")
                if a%3==0:
                    print("**",end=" ")   
                if a%2==0:
                    print("**",end=" ")        
        print(i,end=" ")
    print(end="\n")

1 1 1 1 1 1 1 
2 2 2 2 2 2 2 
3 3 3 3 3 3 3 
4 4 4 4 4 4 4 
5 5 5 5 5 5 5 
